content/source/docs/enterprise/private/preflight-installer.html.md

```diff
@@ -155,14 +155,15 @@ When Terraform Enterprise uses an external PostgreSQL database, the
 following must be present on it:
 
 * PostgreSQL version 9.4 or greater
-* User with all privileges granted on the schemas created and the ability to run "CREATE EXTENSION" on the database
```

Comment:
I might rather we stick with this wording, and only add the sub-bullet with the slightly stronger wording suggested

Reply:
But it's not really true given the other statements. I don't want to contradict this statement just a few lines later.